KIP7 solidity implementation (#227)
klaytn/caver-js: packages/caver-klay/caver-klay-kct/contract/README.md

caver-klay-kct/contract
=================

> caver-klay-kct/contract refers to the contract implementataion in [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).

Reviewer: ```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).
```

caver-klay-kct/contract
=================

> Some files in `caver-klay-kct/contract` were derived from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).

Reviewer 2: Could you please write the version of openzeppelin code?

Reviewer 2: Refer this.
https://github.com/klaytn/klaytn/tree/dev/contracts/externals

Author: @KimKyungup Yes, I wrote the version together. :)

> Some files in `caver-klay-kct/contract` were derived from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).

Author: ```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0).
```

> Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0).

Author: ```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0-rc.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0-rc.0).
```

> Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0-rc.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0-rc.0).

Author: ```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.3.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.3.0).
```
